src/components/layout/ForgotPassWordForm.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styles from './ForgotPasswordForm.module.css'; // Adjust to your path
+import CloseIcon from '../icons/CloseIcon';
+const ForgotPassword = ({ switchForm }) => {
+  console.log('This is the switchform', switchForm);
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.formContainer}>
```

Comment:
I would suggest using an actual form element for a form instead of a div for best practice.